components/gantry/oneaxis/oneaxis.go

```diff
@@ -299,8 +299,9 @@ func (g *oneAxis) testLimit(ctx context.Context, zero bool) (float64, error) {
 			return 0, ctx.Err()
 		}
 	}
-
-	return g.motor.Position(ctx, nil)
+	position, err := g.motor.Position(ctx, nil)
+	time.Sleep(250 * time.Millisecond)
```

Comment:
Would it make sense to swap these lines, so we let the motor settle before getting its position? I can go either way. If you do swap them, we can just return things directly instead of storing into variables and returning those.

Reply:
I do still think it's important to put in a comment explaining why we're sleeping. Otherwise, I think it looks like this is left over from some kind of debugging adventure and should be removed again.

Reply:
I've added a comment it will be there in the next commit